keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -142,6 +144,20 @@ class ClusterHandler(
       resource.spec.resolveCapacity(serverGroup.location.region)
     ) == true
 
```

Comment:
You have to tag amis by name and not id, so we have to make an additional call to clouddriver to get the info about the running ami.